src/containers/ProgressTrackerContainer.jsx

```diff
@@ -140,6 +162,11 @@ ProgressTrackerContainer.propTypes = {
   engrTotal: PropTypes.number.isRequired,
   sciTotal: PropTypes.number.isRequired,
   ahseTotal: PropTypes.number.isRequired,
+  mathSciPerc: PropTypes.string.isRequired,
```

Comment:
"Perc"onally, I am not a fan of this naming scheme. pct is a better abbreviation for percent, and I like whole words written up. But it seems like it'd be a real pain in the ass to change, so we can probably leave it for now.